relayer/spy_relayer/src/__tests__/testUtils.ts

```diff
@@ -81,19 +81,21 @@ export function encodeSwimPayload(
   targetChainRecipient: Buffer,
   propellerEnabled: boolean | null,
   gasKickstartEnabled: boolean | null,
+  maxSwimUSDFee: bigint | null,
   swimTokenNumber: number | null,
   memoId: Buffer | null
 ) {
-  const encoded = Buffer.alloc(53);
+  const encoded = Buffer.alloc(61);
```

Comment:
I implemented something similar here:
https://github.com/swim-io/swim/blob/master/packages/evm-contracts/src/payloads.ts

it's also still messy because you still have to specify the same thing twice (i.e. specify the type that you read and write in the function name and then separately give the size) but at least with my implementation you don't have to specify the offset "manually" anymore, but only have to ensure that type and its size match.

Reply:
This is a good thing to refactor later, will add to the list